bin/model_generator.dart

```diff
@@ -114,6 +117,26 @@ void writeToFiles(
         extendsModelfields,
         modelGeneratorConfig,
       ).write();
+      if (model.generateTable == true) {
+        for (final field
+            in model.fields.where((element) => !element.ignoreForTable)) {
+          final fieldModel = modelGeneratorConfig.models.firstWhereOrNull(
+              (element) =>
+                  element.name == field.type.name ||
+                  (field.type is MapType &&
+                      element.name == (field.type as MapType).valueName));
```

Comment:
What about lists? Sets? Keys as enum?

Reply:
Lists, sets have the type inside type.name, but actually since we don't support maps atm in drift tables maps should not be added here